rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -253,6 +291,13 @@ private PingPeerMessage sendPing(InetSocketAddress nodeAddress, int attempt) {
     synchronized PingPeerMessage sendPing(InetSocketAddress nodeAddress, int attempt, Node node) {
         PingPeerMessage nodeMessage = checkPendingPeerToAddress(nodeAddress);
 
+        logger.debug("sendPing - Sending ping message to " +
+                        "nodeAddress hostName: [{}], " +
+                        "nodeAddress port: [{}], " +
+                        "attempt: [{}], " +
+                        "nodeMessage: [{}]"
+                , nodeAddress.getHostName(), nodeAddress.getPort(), attempt, nodeMessage);
+
```

Comment:
what about printing this a bit below together with - `logger.debug("sendPing - nodeMessage created: [{}]", nodeMessage);` plus printing a debug log inside the `if (nodeMessage != null) {` block? - so that we combine two logs into one. It seems to me that `nodeMessage != null` should be quite rare case

Reply:
Done